Minio/ApiEndpoints/ObjectOperations.cs

```diff
@@ -331,6 +331,58 @@ public async Task<Tagging> GetObjectTagsAsync(GetObjectTagsArgs args, Cancellati
         }
 
 
+        /// <summary>
+        /// Removes an object with given name in specific bucket
+        /// </summary>
+        /// <param name="args">RemoveObjectArgs Arguments Object encapsulates information like - bucket name, object name, optional list of versions to be deleted</param>
+        /// <param name="cancellationToken">Optional cancellation token to cancel the operation</param>
+        /// <returns>Task</returns>
+        public async Task RemoveObjectAsync(RemoveObjectArgs args, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            args.Validate();
+            RestRequest request =  null;
+            if (string.IsNullOrEmpty(args.VersionId))
+            {
+                request = await this.CreateRequest(Method.DELETE, args.BucketName, args.ObjectName).ConfigureAwait(false);
+            }
```

Comment:
can this not be handled by this.CreateRequest(args)

Reply:
Corrected the previous oversight. Also tested.